### ISSUE TITLE
Renderer addCommand problem

### ISSUE THREAD

**Issue by reasonMix** [cocos2d/cocos2d-x#16152]
Renderer hold the command with a c pointer,if a node add the command to the render,and the render havnt do the render,and the node is releaseed,so the command pointer is wildpointer

**Comment by reasonMix** [comment]
this will crashed on ios and android platform

**Comment by dumganhar** [comment]
Yep, it seems to be a problem. Did you invoke `node->visit()` manually?

**Comment by reasonMix** [comment]
use the list view and call pushbackdefaultItem, i want to get the innert container size in current frame ,so i call the visit fucntion

**Comment by dumganhar** [comment]
@zilongshanren , could you check the UI problem in this issue?
Is there a better approach for getting inner container size?
